src/main/java/io/turntabl/employementprofilingsystem/Controllers/RequestController.java

```diff
@@ -0,0 +1,62 @@
+package io.turntabl.employementprofilingsystem.Controllers;
+
+import io.swagger.annotations.ApiOperation;
+import io.turntabl.employementprofilingsystem.Models.RequestTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.*;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.text.SimpleDateFormat;
+import java.util.List;
+
+@RestController
+public class RequestController {
+
+    @Autowired
+    JdbcTemplate jdbcTemplate;
+
+    @CrossOrigin
+    @ApiOperation("Make a request")
+    @PostMapping("/api/v1/request")
+    public void makeARequest(@RequestBody RequestTO request) {
+        jdbcTemplate.update("insert into requests(requester_id, request_start_date, request_report_date) values(?,?,?)",
+                request.getRequester_id(), request.getRequest_start_date(), request.getRequest_report_date());
+    }
+
+    @CrossOrigin
+    @ApiOperation("Get all requests for requester")
+    @GetMapping("/api/v1/request/requester/{id}")
+    public List<RequestTO> getRequestByRequesterId(@PathVariable("id") Integer id) {
+        return this.jdbcTemplate.query(
+                "select * from requests where requester_id = ?",
```

Comment:
don't do select *. What happens if someone comes along later and inserts or removes a column?